### PR TITLE
Add documentation for new CallRecord fields and enums

### PR DESCRIPTION
**Pull request by maperend**
New CallRecord fields: AudioCodec, VideoCodec, NetworkTransportProtocol, TraceRouteHops.
New enums: AudioCodec, VideoCodec, NetworkTransportProtocol.

### CODE REVIEW

File: api-reference/beta/resources/callrecords-traceroutehop.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+---
+title: "traceRouteHop resource type"
+description: "The traceRouteHop type"
+ms.localizationpriority: medium
+author: "maperend"
```
Comment (stephenjust) [inline api-reference/beta/resources/callrecords-traceroutehop.md:5]: You should put a PM's name here, otherwise all bugs will get routed to you.

**Review by stephenjust — APPROVED** [review]
Looks good. I assume the changelog is pending.

File: api-reference/beta/resources/callrecords-traceroutehop.md (hunk 1 above)
Comment (maperend) [reply api-reference/beta/resources/callrecords-traceroutehop.md]: williamlooney

File: api-reference/beta/resources/callrecords-traceroutehop.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+doc_type: "resourcePageType"
+---
+
+# traceRouteHop resource type
+
+Namespace: microsoft.graph.callRecords
+
+[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
+
+Represents the network trace route hops collected for this MediaStream.
```
Comment (JarbasHorst) [inline api-reference/beta/resources/callrecords-traceroutehop.md:16]: If media stream is related to the media stream resource type (api-reference/beta/resources/callrecords-mediastream.md), please add reference link to "media stream".

Same here: api-reference/v1.0/resources/callrecords-traceroutehop.md

**Comment by JarbasHorst** [comment]
Hello @maperend, I reviewed your PR and left one comment. Please, take a look when you will have time.

Also, to avoid merge conflicts, please merge PRs [1](https://github.com/maperend/microsoft-graph-docs/pull/1), [2](https://github.com/maperend/microsoft-graph-docs/pull/2), [3](https://github.com/maperend/microsoft-graph-docs/pull/3), [4](https://github.com/maperend/microsoft-graph-docs/pull/4), [5](https://github.com/maperend/microsoft-graph-docs/pull/5), and [6](https://github.com/maperend/microsoft-graph-docs/pull/6) into your branch to include my fixes and before addressing my comment.

File: api-reference/beta/resources/callrecords-traceroutehop.md (hunk 2 above)
Comment (maperend) [reply api-reference/beta/resources/callrecords-traceroutehop.md]: Reference to media stream resource added.

**Comment by JarbasHorst** [comment]
@Lauragra this PR is ready for final review;

**Review by Lauragra — APPROVED** [review]

**Pull request merged**